go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -326,42 +251,17 @@ func (ovn *Controller) deleteLoadBalancerRejectACL(lb, vip string) {
 	ovn.removeServiceACL(lb, vip)
 }
 
-func (ovn *Controller) findStaleRejectACL(lb, ip string, port int32) (string, error) {
-	aclName := generateACLNameForOVNCommand(lb, ip, port)
-	aclUUID, stderr, err := util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "acl",
-		fmt.Sprintf("name=%s", aclName))
-	if err != nil {
-		klog.Errorf("Error while querying ACLs by name: %s, %v", stderr, err)
-		return "", err
-	} else if len(aclUUID) == 0 {
-		klog.Infof("Reject ACL not found to remove for name: %s", aclName)
-		return "", fmt.Errorf("reject ACL not found to remove for name: %s", aclName)
-	}
-	return aclUUID, nil
-}
-
 // Remove the ACL uuid entry from Logical Switch acl's list.
 func (ovn *Controller) removeACLFromNodeSwitches(switches []string, aclUUID string) {
-	args := []string{}
-	for _, ls := range switches {
-		args = append(args, "--", "--if-exists", "remove", "logical_switch", ls, "acl", aclUUID)
-	}
-
-	if len(args) > 0 {
-		_, _, err := util.RunOVNNbctl(args...)
-		if err != nil {
-			klog.Errorf("Error while removing ACL: %s, from switches, error: %v", aclUUID, err)
-		} else {
-			klog.Infof("ACL: %s, removed from switches: %s", aclUUID, switches)
-		}
+	err := acl.RemoveACLFromNodeSwitches(switches, aclUUID)
```

Comment:
Nit: can't we just remove this function? Anything calling `removeACLFromNodeSwitches` could just do `acl.RemoveACLFromNodeSwitches` instead right?

Reply:
I´m trying to do a smooth transition, once it soak is a simple PR to cleanup all these things ... but it is much more difficult if we do everything at once